integration/makegenesis/genesis.go

```diff
@@ -128,7 +128,7 @@ func NewGenesisBuilder() *GenesisBuilder {
 	if err != nil {
 		panic(fmt.Errorf("failed to create carmen state; %s", err))
 	}
-	carmenStateDb := carmen.CreateStateDBUsing(carmenState)
+	carmenStateDb := carmen.CreateCustomStateDBUsing(carmenState, stateDbCacheCapacity)
```

Comment:
nit: I guess for this case a hard-coded small value of `1024` would have been sufficient, since the scope of creating the genesis is very small. This would have saved the need for forwarding the flag to this point.